stemcell_builder/stages/blobstore_clis/assets/bosh-blobstore-azure-storage.version

```diff
@@ -0,0 +1 @@
+0.0.7
```

Comment:
Just noticed That there is a 0.0.8 already

Reply:
this should auto update later on when the https://github.com/cloudfoundry/bosh-stemcells-ci/pull/14 pr is merged